components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -62,6 +63,8 @@ def main(argv):
         validate_and_load_db_credentials_file(clp_config, clp_home, True)
         validate_and_load_queue_credentials_file(clp_config, clp_home, True)
         validate_and_load_redis_credentials_file(clp_config, clp_home, True)
```

Comment:
i believe those calls should be conditional as well? like queue and redis can be null objects now